extensions/wikia/Wall/WallThread.class.php

```diff
@@ -74,6 +74,8 @@ private function loadReplyObjs() {
 		if ( empty( $this->data->threadReplyIds ) ) {
 			$this->data->threadReplyIds = [ ];
 		}
+		
+		$something = '';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$something" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1481)